src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -1853,6 +1853,23 @@ public void ConnectCallback_GetSet_Roundtrips()
             }
         }
 
+        [Fact]
+        public void PlaintextStreamFilter_GetSet_Roundtrips()
+        {
+            using (var handler = new SocketsHttpHandler())
+            {
+                Assert.Null(handler.PlaintextStreamFilter);
+
+                Func<SocketsHttpPlaintextStreamFilterContext, CancellationToken, ValueTask<Stream>> f = (context, token) => default;
```

Comment:
(You asked about how you could return a null stream even though the return is a non-nullable stream... returning default like this is a good example ;-)